FarihaIS/VanRoomies
# feat(login token + userId) Integrate Login Token Authentication, Track UserId, and Fix Image Size

# Welcome to VanRoomies! 👋✈️

Fixes: #47 
Fixes: #61

## Description of the change:
Integration with token authentication (disabled for M4), tracking of userId using SharedPreferences, and fixes the image size issue. 

## How to manually test:
1. *Run `npm run dev`*
2. Ensure your Google account is not already registered in your local DB
3. Uninstall and Reinstall the app (to force a new sign-in)
4. Log-in
5. Fill out your user preferences (they should be validated by the hints in the form)
6. Try editing your preferences (Existing values should be pre-filled)
7. Go to Listings and get recommended/owned listings
8. Create a new listing!

- The userId should be consistent through each step
- Trying to go to recommended listings before editing preferences should return an empty list
